archiver.go

```diff
@@ -104,7 +111,11 @@ func (tar *ArchiveTar) Unpack(destination string) error {
 
 	if compression, ok := tar.options["tarcompression"]; ok {
 		if unpackTarOpt := tarOptions(compression.(string)); len(unpackTarOpt) > 0 {
-			command = append(command, unpackTarOpt)
+			if usePigz == true {
+				command = append(command, "--use-compress-program=pigz")
```

Comment:
Do we not need to add `-z` to tar command when using pigz then ?

Reply:
It is not required. When `-z` is added to tar command when using pigz, we see the below error,
`tar: Conflicting compression options`